### ISSUE TITLE
Disallowed functions are allowed in any namespace (besides the global one) when neither `use`d nor qualified.

### ISSUE THREAD

**Issue by ZebulanStanphill** [spaze/phpstan-disallowed-calls#124]
Disallowed functions are actually not disallowed at all when they're used in somewhere without being explicitly `use`d or qualified.

```neon
# In PHPStan config...
includes:
	- vendor/spaze/phpstan-disallowed-calls/disallowed-insecure-calls.neon
```

```php
<?php declare(strict_types=1);
// This is in the global namespace.

$x = \md5('blah'); // This is reported as expected.
```

```php
<?php declare(strict_types=1);
namespace My\Namespace;

$x = \md5('blah'); // This is reported as expected.
$y = md5('blah'); // This SHOULD be reported, but isn't!
```

This regression was introduced in [2.6.0](https://github.com/spaze/phpstan-disallowed-calls/releases/tag/v2.6.0) - presumably as an unintended side-effect of #119.

**Comment by spaze** [comment]
Thanks for the report. Fixed and released in [2.7.0](https://github.com/spaze/phpstan-disallowed-calls/releases/tag/v2.7.0). Hopefully no other regression introduced 😊
